src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -0,0 +1,209 @@
+import Title from 'view/title';
+import TizenControlbar from 'view/controls/tizen/tizen-controlbar';
+import DisplayContainer from 'view/controls/display-container';
+import NextUpToolTip from 'view/controls/nextuptooltip';
+import SettingsMenu from 'view/controls/components/menu/settings-menu.js';
+import { addClass } from 'utils/dom';
+import { STATE_PLAYING, STATE_PAUSED } from 'events/events';
+import type ViewModel from 'view/view-model';
+import type { PlayerAPI } from 'types/generic.type';
+
+const Controls = require('../controls').default;
```

Comment:
what's the reasoning behind this syntax?

Reply:
Controls uses `require()` because there was an issue when using `import`. Because we resolve the import file paths by checking commercial first and then os, Controls was being imported as CommercialControls, which was then overwriting the TizenControls. Looking into some workarounds, though this may be resolved when controls get typed.

Reply:
Typing won't change import paths. I think the problem is that CommercialControls is not a class so you can't use it as a base for extending.

Why not make CommercialControls a class so that it can extend this class? It does have some functionality related to ads skip button UI that we may need in certain scenarios.

Reply:
I definitely should have been more clear about typing. What I meant by that was when I had started typing Controls before, I had thought about creating an abstract BaseControls class that could be used by new controls components. So, in this case, TizenControls would import/extend BaseControls as opposed to importing/extending Controls, which I think might solve the issue.

But, from what I could tell, I think the larger issue with this is that CommercialControls instantiates a Controls component.
https://github.com/jwplayer/jwplayer-commercial/blob/master/src/js/view/controls/controls.js#L12
This is what was overriding the TizenControls. Instead of explicitly using the OS Controls here, I tried using the controls component set by the controls-loader (ControlsLoader.controls) and it creates a kind of circular dependency for Tizen. 
TizenControls extends/imports Controls (which is loaded as CommercialControls) and then CommercialControls creates a new instance of TizenControls.
I think converting CommercialControls to a class could possibly result in the same issue since CommercialControls would extend TizenControls and TizenControls would extend the imported CommercialControls.

I'm also curious as to how CommercialControls could be loaded because it seems like importing/using them via OS Controls causes a sort of circular dependency. But, if TizenControls doesn't use OS Controls, CommercialControls wouldn't be loaded from the path resolution. Maybe a commercial controls-loader would be needed and CommercialControls would become like a decorator?

Reply:
Converting CommercialControls to a class resolved this! @robwalch 🎉